partner_contact_department/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Contact department
+==================
+
+This module extends the functionality of the address book to support
+departments.
+
+Department is a drop-down field in partner forms, and it refers to contact
+department in its own company.
+
+Configuration
+=============
+
+To configure departments, you need to:
+
+* Go to *Sales > Configuration > Address Book > Departments*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to any partner's form.
```

Comment:
Maybe not. You are the one that always ask for detailed explanations. Saying "You'll find the field below Job position" is enough.